plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1360,7 +1360,11 @@ public String toString() {
 
             if (_features != null) {
                 for (String feature : _features) {
-                    modeBuilder.append("<feature policy='require' name='" + feature + "'/>");
+                    if (feature.startsWith("-")) {
```

Comment:
minor suggestion - trim to remove accidental spaces if we are checking prefix.

Reply:
@anuragaw 
Thanks for your suggestion.
I checked the code just now. The condition you mentioned is processed in LibvirtComputingResource.java
```
        final String cpuFeatures = (String)params.get("guest.cpu.features");
        if (cpuFeatures != null) {
            _cpuFeatures = new ArrayList<String>();
            for (final String feature: cpuFeatures.split(" ")) {
                if (!feature.isEmpty()) {
                    _cpuFeatures.add(feature);
                }
            }
        }
```